game-core/src/main/java/games/strategy/triplea/settings/ClientSettingSwingUiBinding.java

```diff
@@ -228,6 +228,16 @@
     }
   },
 
+  TEST_HTTP_LOBBY_URI(
+      "Http Lobby URI",
```

Comment:
We should probably consistently display acronyms to the user.  In this one block, we have three different styles: `Http`, `URI`, and `http`.

Reply:
Easy enough to fix: ac2395383

Side-topic: I somewhat wonder if we should develop a code name for the http lobby, so we can call it by something else? The old lobby might be well named 'sanjuro' to match one of its earliest names.

Reply:
Just as long as the code names have an A&A flair to them. :smile: